.ci/windows/Dockerfile

```diff
@@ -13,8 +13,15 @@ RUN Set-ExecutionPolicy Bypass -Scope Process -Force; \
 RUN choco install -y --no-progress visualstudio2022buildtools
 RUN choco install -y --no-progress visualstudio2022-workload-vctools
 
-# Install Git and OpenSSL.
+# Install and OpenSSL and Python.
```

Comment:
Isn't `msysgit` still git; or why did you remove the git comment?

Reply:
No idea what went wrong here. Fixed. :)